### ISSUE TITLE
 Could you please provide your log.txt?

### ISSUE THREAD

**Issue by sunshehai** [yhBeta/Multi-Level-Alignment-Network-for-Cross-Domain-Ship-Detection#2]
Thank you for your work. I feel that there is something wrong with my training results. Could you please provide your log.txt of training and testing?

**Comment by yhBeta** [comment]
Sorry for missing your issues for so long, I'm on an official trip until 18th and all the data are stored in my lab. So I am so sorry that I can't provide you the details until I return.

**Comment by sunshehai** [comment]
> Sorry for missing your issues for so long, I'm on an official trip until 18th and all the data are stored in my lab. So I am so sorry that I can't provide you the details until I return.

Thank you for your reply. First of all, I see that the training data of the target domain in your code is the same as the test data. Shouldn't it be distinguished according to theory? Secondly, I used the same data without any other changes and found that the accuracy was higher than the result in your paper. 150k: 68.51, 140k: 67.61, 120k: 67.29, 100k: 62.36, 80k: 59.77, 60k: 66.32， whether to distinguish the target domain to obtain accuracy than high map in the paper(resnet50 hrrsd to ssdd). In addition, may I ask how to set the iteration number 120k in your paper? The loss in log.txt does not show any rules.

**Comment by yhBeta** [comment]
> 

To be honest, I am not the author of this paper. The author graduated last year. ​I am a newbie master in this research group. I'm just responsible for uploading the code. I haven't even met the author! So I'm very sorry, if you need data source or training logs, I can provide them to you through Baidu cloud disk. These days my supervisor will resign and jump to another college, my own work is in a mess，​so the in-depth research of this paper is at risk, just for your reference.​​​​

**Comment by yhBeta** [comment]
> 

To be honest, I am not the author of this paper. The author graduated last year. ​I am a newbie master in this research group. I'm just responsible for uploading the code. I haven't even met the author! So I'm very sorry, if you need data source or training logs, I can provide them to you through Baidu cloud disk. These days my supervisor will resign and jump to another college, my own work is in a mess，​so the in-depth research of this paper is at risk, just for your reference.​​​​

**Comment by sunshehai** [comment]
Thank your reply, could you provide the training logs?



发自我的iPhone


------------------ Original ------------------
From: YHBeta ***@***.***&gt;
Date: Sun,Mar 26,2023 10:07 PM
To: yhBeta/Multi-Level-Alignment-Network-for-Cross-Domain-Ship-Detection ***@***.***&gt;
Cc: sunshehai ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: Re:[yhBeta/Multi-Level-Alignment-Network-for-Cross-Domain-Ship-Detection]  Couldyou please provide your log.txt? (Issue #2)





   
To be honest, I am not the author of this paper. The author graduated last year. ​I am a newbie master in this research group. I'm just responsible for uploading the code. I haven't even met the author! So I'm very sorry, if you need data source or training logs, I can provide them to you through Baidu cloud disk. These days my supervisor will resign and jump to another college, my own work is in a mess，​so the in-depth research of this paper is at risk, just for your reference.​​​​

**Comment by yhBeta** [comment]
I'm not sure if the training logs are still on the server, I'm going to find it tomorrow morning.

**Comment by yhBeta** [comment]
Here is the only one training log:
https://pan.baidu.com/s/1zhqUtapGE7PZHnbPVkckGg?pwd=v5i1 
Extraction code：v5i1
